nvidia/nvidia-container-toolkit

```diff
@@ -4,37 +4,27 @@ set -eu
 
 . "${SNAP}/usr/share/nvidia-container-toolkit/lib"
 
-# Just exit if NVIDIA support is disabled #
-[ "${NVIDIA_SUPPORT_DISABLED}" != "true" ] || exit 0
+[ "${NVIDIA_SUPPORT_CLASSIC}" == "true" ] && echo "Running on Classic system"
+[ "${NVIDIA_SUPPORT_CORE}" == "true" ] && echo "Running on Ubuntu Core system"
 
-NVIDIA_ENABLE="false"
+# FIXME: error here if both core and classic are detecgted ?
+# [ "${NVIDIA_SUPPORT_CLASSIC}" == "true" -a "${NVIDIA_SUPPORT_CORE}" == "true"  ]
```

Comment:
Refact: Log classic vs core, else error. 
Shouldn't this be done in system-detection script?